02-learning-objectives.Rmd

```diff
@@ -0,0 +1,3 @@
+## Learning Objectives
```

Comment:
I wonder if it's just worth deleting [02-learning-objectives.Rmd](https://github.com/jhudsl/GDSCN_Book_Statistics_for_Genomics_PCA/pull/1/files#diff-6790407793effcdf83c03c1b7bcdd4efaf17a0c02db91faeda47150a19fa8b93) and [03-lesson-plan.Rmd](https://github.com/jhudsl/GDSCN_Book_Statistics_for_Genomics_PCA/pull/1/files#diff-ced49eb84558c21f7b7a177ec729c4b2c0543edbe723f6072565f91f9403b38d) since they don't really have any info yet? You could just drop them from [_bookdown.yml](https://github.com/jhudsl/GDSCN_Book_Statistics_for_Genomics_PCA/pull/1/files#diff-42a58381e95452b8042ac17313c61d81eefca342ccba66b50fe8af8b5bf72d98).. I don't feel terribly strongly about this, so happy to do whatever you prefer!

Reply:
I didn't realize I could just leave them out, tbh. Their lack of content kind of taunts me. I am happy to remove them for now if you don't think it's a problem the format is slightly different from the other books.